src/liboslexec/constfold.cpp

```diff
@@ -2727,19 +2708,15 @@ DECLFOLDER(constfold_pointcloud_search)
         int const_array_sym = rop.add_constant (const_valtype, const_data);
         // ... and add an instruction to copy the constant into the
         // original destination for the query.
-        std::vector<int> args_to_add;
-        args_to_add.push_back (value_args[i]);
-        args_to_add.push_back (const_array_sym);
+        const int args_to_add[2] = { value_args[i], const_array_sym };
```

Comment:
Not critical, but would you like this idiom more, or less, if this were

    cost int args_to_add[] = { ... }

(no magic numeral at all)???

Reply:
Yeah I guess that `2` is a bit redundant. I can update it.

Reply:
I'm just thinking, it's one less thing that can go wrong if at a later point args are added or deleted, or things are cut and pasted around.